glean-core/src/lib.rs

```diff
@@ -65,6 +65,8 @@ pub struct Configuration {
     pub application_id: String,
     /// The maximum number of events to store before sending a ping containing events.
     pub max_events: Option<usize>,
+    /// Whether Glean should defer ping collection.
```

Comment:
Can you please expand on what this implies? Please note that this does not defer "ping" collection, but it defers persisting to disk metrics that have "lifetime: ping".

Reply:
Ah, sorry this one slipped my mind.

What do you think about "Whether Glean should delay persistence of metrics with 'ping' lifetime data."

Reply:
No problem :) Maybe "Whether Glean should delay persistence of data from metrics with `ping` lifetime"?